config/settings.go

```diff
@@ -1,8 +1,9 @@
 package config
 
 type Settings struct {
-	DefaultPort    int    `hcl:"default_port,optional"`
-	XForwardedHost bool   `hcl:"xfh,optional"`
-	HealthPath     string `hcl:"health_path,optional"`
-	LogFormat      string `hcl:"log_format,optional"`
+	DefaultPort     int    `hcl:"default_port,optional"`
+	XForwardedHost  bool   `hcl:"xfh,optional"`
+	HealthPath      string `hcl:"health_path,optional"`
+	LogFormat       string `hcl:"log_format,optional"`
+	RequestIDFormat string `hcl:"request_id_format,optional"`
```

Comment:
The added field is currently not referenced but should be mapped to `HTTPConfig`.